README.md

```diff
@@ -26,77 +26,128 @@ The Pulsar Node.js client can be used to create Pulsar producers and consumers i
 This library works only in Node.js 10.x or later because it uses the
 [node-addon-api](https://github.com/nodejs/node-addon-api) module to wrap the C++ library.
 
-## How to install
+## Getting Started
 
 > **Note**
 >
 > These instructions are only available for versions after 1.8.0. For versions previous to 1.8.0, you need to install the C++ client first. Please switch to the corresponding version branch of this repo to read the specific instructions.
+>
+> To run the examples, skip this section.
```

Comment:
I kept this part because currently we cannot run `npm install pulsar-client` without installing the C++ client, see https://github.com/apache/pulsar-client-node/issues/254. After the new release, we should remove this note.

/cc @shibd @RobertIndie @tisonkun

Reply:
+1